Tools/Scripts/libraries/reporelaypy/run

```diff
@@ -188,8 +188,11 @@ def main(args=None):
 
     processor = HookProcessor(checkout=checkout, database=database) if args.hooks else None
 
+    import gunicorn
+    gunicorn_path = os.path.abspath(os.path.join(gunicorn.__path__[0], '../../../../bin/gunicorn'))
```

Comment:
This terrifies me, hardcoding a path like this, because that is _incredibly_ fragile.

We expect it to be installed as a dependency of `webkitflaskpy`, I believe, because we don't seem to have it autoinstalled anywhere:

https://github.com/WebKit/WebKit/blob/f52abbbacd4b9fd731e8a250e5e8c6379add50a5/Tools/Scripts/libraries/webkitflaskpy/setup.py#L60

If we want to find the actual path to the script, `shutil.which("gunicorn", sysconfig.get_path("scripts"))` should find it, assuming we're not in a `venv` which has access to the system `site-packages`.

Alternatively, since https://github.com/benoitc/gunicorn/commit/17e70dac536f182f8aa00f5e22bfed5985661081 we could just do `[sys.executable, "-m", "gunicorn"]`, which is probably the better option?